website/docs/r/database_cluster.html.markdown

```diff
@@ -52,11 +52,12 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the database cluster.
 * `engine` - (Required) Database engine used by the cluster (ex. `pg` for PostreSQL, `mysql` for MySQL, or `redis` for Redis).
-* `size` - (Required) Database droplet size associated with the cluster (ex. `db-s-1vcpu-1gb`).
+* `size` - (Required) Database Droplet size associated with the cluster (ex. `db-s-1vcpu-1gb`).
 * `region` - (Required) DigitalOcean region where the cluster will reside.
 * `node_count` - (Required) Number of nodes that will be included in the cluster.
 * `version` - (Optional) Engine version used by the cluster (ex. `11` for PostgreSQL 11).
 * `tags` - (Optional) A list of tag names to be applied to the database cluster.
+* `eviction_policy` - (Optional) A string specifying the eviction policy for a Redis cluster. Valid values are: `noeviction`, `allkeys_lru`, `allkeys_random`, `volatile_lru`, `volatile_random`, or `volatile_ttl`.
```

Comment:
I would love to see the valid values autopopulated: I'm always concerned that manually maintained values in docs could get out of date. Better to have the manually maintained values than not have them at all.